app/src/components/v-upload.vue

```diff
@@ -146,10 +154,12 @@ function useUpload() {
 		if (props.folder) {
 			folderPreset.folder = props.folder;
 		}
-
 		try {
-			numberOfFiles.value = files.length;
+			if (!validFiles(files)) {
+				throw new Error('An error has occurred while uploading the files.');
```

Comment:
Do we want to be slightly more explicit here? Just in case any users might keep retrying but not sure what happened.

Reply:
Was considering the same but im afraid this may not be the only situation that could trigger these conditions (even though im not explicitly aware of any) 🤔 So for now i chose the generic message but happy to hear any alternatives

Reply:
There aren't really more errors to throw i think 🤔 I currently see it as either abstract "an error in uploading has occured" or specificly the issue we're covering "No data to upload. If you're directly uploading archived files then unpack them first". The check itself is not so specific though and possibly could trigger on more upload error situations like file read errors etc.